### PR TITLE
Add a pointer to datalad/shrinky prototype

### PR DESCRIPTION
**Pull request by yarikoptic**
Developed with @Remi-Gau during brainhack. Ideally should be generalized into something even more generic.

### CODE REVIEW

File: kbi/0013/index.rst [hunk 1]
```diff
@@ -131,5 +131,10 @@ Now follow the same steps for ``file2.txt`` and ``target2``::
     $ cat target2/file2.txt
     456
 
+A [datalad/shrinky](https://github.com/datalad/shrinky) provides a reusable
```
Comment (mslw) [inline kbi/0013/index.rst:134]:
```suggestion
A `datalad/shrinky <https://github.com/datalad/shrinky>`_ provides a reusable
```
reStructuredText :wink:

**Review by mslw — APPROVED** [review]
It feels a little bit out of context when the paragraph is right after the last code block, but overall a cool addition.

A link needs a fix though.

File: kbi/0013/index.rst (hunk 1 above)
Comment (yarikoptic) [reply kbi/0013/index.rst]: Thank you @mslw !

**Review by loj — APPROVED** [review]
Nice! Thanks for the addition. :-)

**Pull request merged**